src/main/java/ui/components/pickers/MilestonePickerDialog.java

```diff
@@ -255,6 +248,7 @@ private FlowPane createAssignedMilestoneBox() {
 
     private VBox createMatchingMilestonesBox() {
         VBox milestoneGroup = new VBox();
+        milestoneGroup.setPrefHeight(30 * MilestonePickerState.BEST_MATCHING_LIMIT);
```

Comment:
Is `30` here the same as above (`detailedMilestoneBox.setPrefHeight(30)`)? If yes, we should abstract it out into a constant.